contracts/HATVault.sol

```diff
@@ -176,6 +176,7 @@ contract HATVault is IHATVault, ERC4626Upgradeable, OwnableUpgradeable, Reentran
         if (_params.maxBounty > MAX_BOUNTY_LIMIT)
             revert MaxBountyCannotBeMoreThanMaxBountyLimit();
         _validateSplit(_params.bountySplit);
+        __ERC20_init(string.concat("HATs Vault ", _params.name), string.concat("HAT", _params.symbol));
```

Comment:
```suggestion
        __ERC20_init(string.concat("Hats Vault ", _params.name), string.concat("HAT", _params.name));
```

Reply:
the shares token symbol should also be with the name of the vault and not the name of the vault's token, e.g HATAngle, HATDXDAO, HATICHI.. and not HATUSDC

Reply:
mm, as i understood our decision was to provide two separate arguments, for _symbol and _name (and so we do not change this)

Reply:
I also understood we want to pass name and symbol, we can't really derive the symbol from the name

Reply:
the symbol is HAT+the project name, isn't it?